docs/tau-updates.md

```diff
@@ -0,0 +1,128 @@
+# Targetability Updates (TAU)
+
+In cases where fflogs sends insufficient information to determine the correct information and state of a boss (invulnerability window, ephemeral enemies, etc), there is a mechanism to override this on a per-fight basis under `src/parser/bosses`.
+
+# Setup
+
+## Scaffolding
+
+At current, you'll need to provide the following scaffolding / directory structure:
+
+```
```

Comment:
just thinking;

while this makes sense to keep in-band for now; this part is essentially docs on scaffolding a meta group, rather than invuln specifically. should we ever have two boss-specific override modules, would likely move scaffolding (if still required) into a separate doc.